src/backend/marsha/core/tests/test_api_video.py

```diff
@@ -4678,9 +4678,14 @@ def test_api_video_instructor_end_idle_live(self):
         )
 
     @override_settings(LIVE_CHAT_ENABLED=True)
+    @override_settings(XMPP_BOSH_URL="https://xmpp-server.com/http-bind")
+    @override_settings(XMPP_CONFERENCE_DOMAIN="conference.xmpp-server.com")
+    @override_settings(XMPP_DOMAIN="conference.xmpp-server.com")
+    @override_settings(XMPP_JWT_SHARED_SECRET="xmpp_shared_secret")
     def test_api_video_instructor_end_paused_live(self):
         """An instructor can end a live in paused state"""
         video = factories.VideoFactory(
+            id="1c731481-f919-4491-8be6-84b0b344dd8f",
```

Comment:
Why using a specific id here ?

Reply:
It was for the `xmpp` block in the reponse but I decided to use `video.id` so this change can be reverted